base/shell/explorer/traywnd.cpp

```diff
@@ -1628,14 +1628,21 @@ class CTrayWindow :
         if (StartSize.cx > rcClient.right)
             StartSize.cx = rcClient.right;
 
-        if (!m_Theme)
+        HWND hwndTaskToolbar = ::GetWindow(m_TaskSwitch, GW_CHILD);
+        if (hwndTaskToolbar)
         {
-            HWND hwndTaskToolbar = ::GetWindow(m_TaskSwitch, GW_CHILD);
-            if (hwndTaskToolbar)
+            SIZE EdgeSize;
+            EdgeSize.cy = GetSystemMetrics(SM_CYEDGE);
```

Comment:
```suggestion
            LONG EdgeSizeY = GetSystemMetrics(SM_CYEDGE);
```
This didn't need to be a full SIZE structure if you're just using its cy member.